### ISSUE TITLE
I met a problem when trying to catkin the file

### ISSUE THREAD

**Issue by beforeface** [eborghi10/jps_global_planner#4]
When I failed to use "cd /catkin_ws && catkin_make_isolated --install --install-space /opt/ros/$ROS_DISTRO -DCMAKE_BUILD_TYPE=Release" ,   I do not know how to do it ,the problem is"file cannot create directory:/opt/ros/kinetic/share.

**Comment by eborghi10** [comment]
Hi @beforeface, If you use `catkin_make` as usual it will work well too.

**Comment by beforeface** [comment]
> Hi @beforeface, If you use `catkin_make` as usual it will work well too.

I don't think so.I'm a new guy for ros. I downloaded the jps in a workspace.Using command catkin_make_isolated failed and the problem is what I said before.I tried to use caktin_make,but it said" this workspace contains non-catkin package in it.try the catkin_make_isolated command instead.I don't know how to solve it. So,please help me.

**Comment by eborghi10** [comment]
You have to remove the `build_isolated` and `devel_isolated` directories before running `catkin_make`.

**Comment by beforeface** [comment]
I'm sorry.I can not understand it. In the project page, I run two commands under the "Install dependencies",the first is good, the second failed,it said"CMake Error at cmake_install.cmake:36 (file):
  file cannot create directory: /opt/ros/kinetic/share/jps3d/cmake.  Maybe
  need administrative privileges.

**Comment by beforeface** [comment]
> You have to remove the `build_isolated` and `devel_isolated` directories before running `catkin_make`.

I'm sorry.I can not understand it. In the project page, after I run **two commands under the "Install dependencies"**,the first is good, the second failed,which said"CMake Error at cmake_install.cmake:36 (file):file cannot create directory: /opt/ros/kinetic/share/jps3d/cmake. Maybe need administrative privileges."Then I  do as what you said,remove the **build_isolated** and **devel_isolated**and **use catkin_make**, but it said" CMake Error at /opt/ros/kinetic/share/catkin/cmak/catkin_workspace.cmake:100 (message): This workspace contains non-catkin packages in it, and catkin cannot build
  a non-homogeneous workspace without isolation.  Try the 'catkin_make_isolated' command instead."

**Comment by eborghi10** [comment]
So, based on the first error that you posted, it seems like you don't have ROS installed. What do you have in `/opt/ros`?

**Comment by beforeface** [comment]
> So, based on the first error that you posted, it seems like you don't have ROS installed. What do you have in `/opt/ros`?

Sorry for replying late.I have ROS.I can run my own project.But I tried to replace the plugin,it falied.As what I said before,I don't know how to fix it.
